docs/developer/users/staff-users.mdx

```diff
@@ -0,0 +1,100 @@
+3. Enter the staff member’s first name, last name, and email address.
+4. Click **Send Invite**.
+
+Saleor will send an email invitation to the provided address. The recipient can set their password and log in using a secure link.
+
+After inviting a staff user, you can assign them to a permission group. 
+For details on permission groups, scopes, and channel restrictions, see the [Permissions documentation](/developer/permissions.mdx).
+
+
+### API
+You can also create staff users programmatically using the [`staffCreate`](../../api-reference/users/mutations/staff-create.mdx) mutation 
+This allows you to assign permissions and add metadata in the same step.
+
+```graphql {2-8}
+mutation {
+  staffCreate(input: {
+    email: "team.member@example.com"
+    firstName: "Team"
+    lastName: "Member"
+    addGroups: ["R3JvdXA6MQ=="]  # Optional: Assign to permission group
+  }) {
+    user {
+      id
+      email
+    }
+    errors {
+      field
+      message
+    }
+  }
+}
+```
+
+## Updating Staff Data
+You can update staff user details using the Dashboard or the [`staffUpdate`](../../api-reference/users/mutations/staff-update.mdx) mutation.
+
+## Removing Staff Users
+You can remove staff users using the Dashboard or the API:
+- **Dashboard:** Remove staff users from the **Staff Members** section.
+- **API:** Use the [`staffDelete`](../../api-reference/users/mutations/staff-delete.mdx) or [`staffBulkDelete`](../../api-reference/users/mutations/staff-bulk-delete.mdx) mutations.
+
+:::info
+If a staff user has placed orders, their customer record will remain for order history and reporting. 
+Removing a staff user does not delete their customer data.
+:::
+
+## Extending Staff Users with Metadata
+You can attach custom metadata to staff user accounts for integrations or custom fields.
+
+- To set or update metadata when updating a user, include the `metadata` or `privateMetadata` fields in the [`staffUpdate`](../../api-reference/users/mutations/staff-update.mdx) mutation.
+- For more granular control, use:
+  - [`updateMetadata`](../../api-reference/miscellaneous/mutations/update-metadata.mdx) for public key-value data (visible to apps and staff).
+  - [`updatePrivateMetadata`](../../api-reference/miscellaneous/mutations/update-private-metadata.mdx) for secure/internal values (not exposed via the public API).
+
+## Dashboard Features for Staff Users
+Saleor provides advanced dashboard features to help staff users personalize their workspace and work efficiently.
+
+### Saving Filter Presets
+You can save filter presets on list views for quick access to frequently used filters (e.g., “Out of Stock Products”, “High-Value Orders”).
+
+**To save a filter preset:**
+1. Go to a list view (e.g., Products).
+2. Apply filters (such as channel or search terms).
+3. Click the **+** icon next to the list title.
+4. Enter a name for your preset and click **Save**.
+
+Presets are available only to the staff user who created them.
```

Comment:
Is it always available no matter from where I am logging in? Or stored in local browser storage?